vehicle/OVMS.V3/changes.txt

```diff
@@ -1,13 +1,78 @@
 Open Vehicle Monitor System v3 - Change log
 
 ????-??-?? ???  ???????  OTA release
+- Vehicle framework: Scheduled precondition control with duration support
+    Centralized implementation for all vehicles with per-vehicle override capability.
+    Supports multiple time slots per day with individual duration settings.
+    Global enable/disable switch with config persistence.
+  New commands:
+    climatecontrol schedule set <day> <times>  -- Set schedule for a day, times format: HH:MM[/duration][,HH:MM[/duration],...]
+                                                     Examples: "07:30", "07:30/10", "07:00/5,17:30/15"
+                                                     Duration in minutes
+    climatecontrol schedule list               -- List all configured schedules with next scheduled event
+    climatecontrol schedule clear <day|all>    -- Clear schedule for specific day or all days
+    climatecontrol schedule enable             -- Enable scheduled precondition (global switch)
+    climatecontrol schedule disable            -- Disable scheduled precondition (global switch)
+    climatecontrol schedule status             -- Show schedule status and configured times
+  New config:
+    [vehicle] schedule.<day>               -- Schedule for day (mon/tue/wed/thu/fri/sat/sun)
+                                              Format: HH:MM[/duration][,HH:MM[/duration],...]
+    [vehicle] precondition                 -- Enable/disable scheduled precondition (bool, default false)
```

Comment:
And accordingly `climate.precondition` for this entry, to keep these close when listing, and to allow other preconditioning control (battery for example) later on.